MekHQ/resources/mekhq/resources/PersonViewPanel.properties

```diff
@@ -1,3 +1,9 @@
+# General Properties
+Time.YearYears=year(s)
```

Comment:
YearYears?

Reply:
Couldn't think of a better way to say year(s)